README.md

```diff
@@ -12,29 +12,28 @@ The Operator will also watch all Gitea resources and reinstall them if they are
 
 ## Installing the Operator
 
-First we need to create a Service Account, Role and Role Binding in order to grant the required permissions to the Operator. Make sure you are logged in with a user that has permission to create those resources.
+First we need to create a Service Account, Role and Role Binding in order to grant the required permissions to the Operator. The `install` target of the Makefile will take care of this. Make sure you are logged in with a user that has permission to create those resources.
 
 ```sh
 $ oc login -u system:admin
-$ oc create -f deploy/service_account.yaml
-$ oc create -f deploy/role.yaml
-$ oc create -f deploy/role_binding.yaml
+$ make build
```

Comment:
we might one to specify org here like make build ORG=<Your Quay.io Org> as this defaults to integreatly. same for make push

Reply:
Good point